go/vt/sqlparser/ast_format.go

```diff
@@ -1512,10 +1512,14 @@ func (node *FuncExpr) Format(buf *TrackedBuffer) {
 // Format formats the node
 func (node *GroupConcatExpr) Format(buf *TrackedBuffer) {
 	if node.Distinct {
-		buf.astPrintf(node, "group_concat(%s%v%v%s%v)", DistinctStr, node.Exprs, node.OrderBy, node.Separator, node.Limit)
+		buf.astPrintf(node, "group_concat(%s%v%v", DistinctStr, node.Exprs, node.OrderBy)
 	} else {
-		buf.astPrintf(node, "group_concat(%v%v%s%v)", node.Exprs, node.OrderBy, node.Separator, node.Limit)
+		buf.astPrintf(node, "group_concat(%v%v", node.Exprs, node.OrderBy)
 	}
+	if node.Separator != "" {
```

Comment:
This works because also if the separate is set to `''` explicitly, that means we have that as the encoded string here already so we print it again. 

Before we would treat it combined as one thing, but we'd upcase the whole thing, breaking this.